javascript/javascript1/week2/script.js

```diff
@@ -0,0 +1,167 @@
+// Flight booking full name function =============================================================================
+
+const getFullName = (firstName, lastName) => `${firstName} ${lastName}`;
+
+const fullName1 = getFullName('Rostyslav', 'Zhyvitskyi');
+const fullName2 = getFullName('Benjamin', 'Hughes');
+
+console.log(fullName1, fullName2);
+
+// Formal full name
+
+const getFullNameFormal = (firstName, lastName, sex, useFormalName) => {
+  if (firstName === '' || lastName === '') {
```

Comment:
This can be written as `if (!firstName || !lastName)`

Reply:
and with that having only sex and useFormalName parameters would be solved also, for example `getFullNameFormal('male', false)`

Reply:
Fixed 😊